src/themes/teams/siteVariables.ts

```diff
@@ -87,9 +87,14 @@ export const textLargeFontSize = '18pt'
 export const textLargeLineHeight = '24pt'
 export const textExtraLargeFontSize = '24pt'
 export const textExtraLargeLineHeight = '32pt'
-export const textX2FontSize = ''
-export const textX2LineHeight = ''
-export const textX3FontSize = ''
-export const textX3LineHeight = ''
-export const textX4FontSize = ''
-export const textX4LineHeight = ''
+export const textX2FontSize = '36pt'
+export const textX2LineHeight = '48pt'
+export const textX3FontSize = '54pt'
+export const textX3LineHeight = '66pt'
+export const textX4FontSize = '72pt'
+export const textX4LineHeight = '86pt'
+export const textWeightLight = '200'
+export const textWeightSemilight = '300'
+export const textWeightRegular = '400'
+export const textWeightSemibold = '600'
+export const textWeightBold = '800'
```

Comment:
To prevent typing errors later when we support variable typings in style objects, `fontWeight` values should be numbers opposed to strings.